config/config-observability.yaml

```diff
@@ -82,3 +82,18 @@ data:
   # the kibana dashboard using `kubectl proxy`.
   logging.revision-url-template: |
     http://localhost:8001/api/v1/namespaces/knative-monitoring/services/kibana-logging/proxy/app/kibana#/discover?_a=(query:(match:(kubernetes.labels.knative-dev%2FrevisionUID:(query:'${REVISION_UID}',type:phrase))))
+
+# METRICS CONFIGURATION
+
+  # metrics.backend-destination field specifies the system metrics destination.
+  # It defaults to prometheus. If this is stackdriver, the metrics will be sent
+  # to stackdriver. "prometheus" and "stackdriver" are supported. "prometheus"
+  # will be used for any other values. This field is required.
+  metrics.backend-destination: "prometheus"
+
+  # metrics.stackdriver-project-id field specifies the stackdriver project ID.
+  # To export metrics to stackdriver, you need to have a valid stackdriver project ID.
+  # Note: Using stackdriver would incur charges for knative-serving system metrics
+  # before we have own resource types.
+  metrics.stackdriver-project-id: "<your stackdriver project id>" 
```

Comment:
I'd comment this if you are putting in a bogus value.

Reply:
Can we use [this](https://godoc.org/cloud.google.com/go/compute/metadata#ProjectID) to default this when the `backend-destination` is `"stackdriver"`, but `stackdriver-project-id` is omitted?

This is going to be the right answer 99.9% of the time.

Reply:
Thanks for the feedback. We could actually skip this field if it's in GKE. I sent https://github.com/knative/pkg/pull/147 for review.

Reply:
Can we comment this line out as well?

Reply:
Yes, I'll polish this pr after https://github.com/knative/pkg/pull/147 is checked in.

Reply:
Should be in!